packages/sit-onyx/src/components/OnyxForm/OnyxForm.stories.ts

```diff
@@ -0,0 +1,46 @@
+import { withNativeEventLogging } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import { h } from "vue";
+import { OnyxInput } from "../..";
+import OnyxButton from "../OnyxButton/OnyxButton.vue";
+import OnyxStepper from "../OnyxStepper/OnyxStepper.vue";
+import OnyxForm from "./OnyxForm.vue";
+
+/**
+ * This component can be used in to wrap form components like OnyxInput/OnyxTextarea/OnyxStepper.
+ * Currently it can be used to set all form elements to disabled.
```

Comment:
We will most likely not maintain this comment in the future
```suggestion
 * It can be used to e.g. set all form components to disabled.
```